content/zh/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,292 @@
+-->
+拓扑管理器从 *Hint Providers* 接收拓扑信息，作为表示 NUMA 可用节点和首选分配指示的位掩码。
+拓扑管理器策略对提供的提示执行一组操作，并汇集策略确定的提示以提供最佳结果，如果存储了不希望的提示，则该提示的首选字段将设置为 false。
+在当前策略中，首选的是最窄的首选掩码。
+所选提示将存储为拓扑管理器的一部分。
+根据配置的策略，可以根据所选提示从节点接受或拒绝 pod 。
+提示然后存储在拓扑管理器中，供 *Hint Providers* 进行资源分配决策时使用。
+
+<!--
+### Topology Manager Policies
+-->
+### 拓扑管理器策略
+
+<!--
+The Topology Manager currently:
+-->
+当前拓扑管理器：
+
+<!--
+ - Works on Nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/)
+ - Works on Pods in the `Guaranteed` {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}
+-->
+- 在启用了 `静态` CPU 管理器策略的节点上工作。 请参阅 [控制 CPU 管理策略](/docs/tasks/administer-cluster/cpu-management-policies/)
+- 在 `保证` 的 pod 上工作 {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}
+
+<!--
+If these conditions are met, Topology Manager will align CPU and device requests.
+-->
+如果满足这些条件，则拓扑管理器将调整 CPU 和设备请求。
+
+<!--
+Topology Manager supports four allocation policies. You can set a policy via a Kubelet flag, `--topology-manager-policy`.
+There are four supported policies:
+-->
+拓扑管理器支持四种分配策略。
+您可以通过 Kubelet 标志 `--topology-manager-policy` 设置策略。
+有四种受支持的策略：
+
+<!--
+* `none` (default)
+* `best-effort`
+* `restricted`
+* `single-numa-node`
+-->
+* `none` (默认)
+* `best-effort`
+* `限制`
+* `单节点数`
+
+<!--
+### none policy {#policy-none}
+-->
+### none 策略 {#policy-none}
+
+<!--
+This is the default policy and does not perform any topology alignment.
+-->
+这是默认策略，不执行任何拓扑对齐。
+
+<!--
+### best-effort policy {#policy-best-effort}
+-->
+### best-effort 策略 {#policy-best-effort}
+
+<!--
+For each container in a Guaranteed Pod, kubelet, with `best-effort` topology 
+management policy, calls each Hint Provider to discover their resource availability.
+Using this information, the Topology Manager stores the 
+preferred NUMA Node affinity for that container. If the affinity is not preferred, 
+Topology Manager will store this and admit the pod to the node anyway.
+-->
+对于有保证的 Pod 中的每个容器，具有 `best-effort` 拓扑管理策略的 kubelet 调用每个提示提供程序以发现其资源可用性。
+使用此信息，拓扑管理器存储该容器的首选 NUMA 节点亲和性。
+如果亲和性不是首选，则拓扑管理器将存储该亲和性，并且无论如何都将  pod 接纳到该节点。
+
+<!--
+The *Hint Providers* can then use this information when making the 
+resource allocation decision.
+-->
+然后 *Hint Providers* 可以在进行资源分配决策时使用这个信息。
+
+<!--
+### restricted policy {#policy-restricted}
+-->
+### 限制策略 {#policy-restricted}
+
+<!--
+For each container in a Guaranteed Pod, kubelet, with `restricted` topology 
+management policy, calls each Hint Provider to discover their resource availability.
+Using this information, the Topology Manager stores the 
+preferred NUMA Node affinity for that container. If the affinity is not preferred, 
+Topology Manager will reject this pod from the node. This will result in a pod in a `Terminated` state with a pod admission failure.
+-->
+对于保证 Pod 中的每个容器， kubelet 具有 `受限` 拓扑管理策略，调用每个提示提供程序以发现其资源可用性。
```

Comment:
```suggestion
对于 Guaranteed 类 Pod 中的每个容器，配置了 `restricted` 拓扑管理策略的 kublet 调用每个建议提供者以确定其资源可用性。
```

Reply:
@tengqm I think there is a typo in that suggestion (“_kublet_”)